.github/workflows/pr_lint_and_test.yml

```diff
@@ -7,12 +7,16 @@ jobs:
     runs-on: ubuntu-latest
     env:
       NODE_ENV: dev
+    strategy:
+      matrix:
+        version: [12, 14, 16]
+      fail-fast: false
     steps:
       - uses: actions/checkout@v3
-      - name: "Use NodeJS 16"
+      - name: "Use NodeJS"
         uses: actions/setup-node@v3
         with:
-          node-version: '16'
+          node-version: ${{ matrix.version }}
       - name: Install npm@8.x
```

Comment:
Some unit tests fail in NodeJS12, so I make use of matrix strategy here to make sure that we don't introduce any issue later.